relnotes/meta-strip-arrays.feature.md

```diff
@@ -0,0 +1,5 @@
+* `ocean.meta.types.Arrays`
+
+  `ocean.core.Traits.BaseTypeOfArrays` was reimplemented as
```

Comment:
"reimplemented" implies that it has the same function. The release note should probably explain why one would use `StripAllArrays` over `BaseTypeOfArrays` (or are both still useful in different situations?).

Reply:
It has same functionality and old one is one day going to be deprecated with all `ocean.core.Traits`. That is true for all utils reimplemented from `ocean.core.Traits` - any specific wording you would like to see?

Reply:
What you added answers my question :+1: